src/item_action.cpp

```diff
@@ -49,15 +51,19 @@ class actmenu_cb : public uimenu_callback {
             }
             // Don't write a message if unknown command was sent
             // Only when an inexistent tool was selected
-            auto itemless_action = item_actions.find( action );
-            if( itemless_action != item_actions.end() ) {
+            auto itemless_action = am.find( action );
+            if( itemless_action != am.end() ) {
                 popup( _("You do not have an item that can perform this action.") );
             }
             return false;
         }
 };
 
-item_action_map map_actions_to_items( player &p )
+item_action_generator::item_action_generator() = default;
+
+item_action_generator::~item_action_generator() = default;
```

Comment:
@kevingranade If you're referencing #11530, that's from some other defaulted destructor -- this one is non-virtual and has nothing to do with the `it_{,artifact}_{tool,armor}` classes.